src/main/madara-app.ts

```diff
@@ -190,7 +190,93 @@ export async function startApp(window: BrowserWindow, appId: string) {
   if (appConfig.appType === 'binary') {
     await startAppBinary(window, appConfig);
   } else if (appConfig.appType === 'docker') {
-    await startAppDocker(window, appConfig);
+    // const containerNames = appConfig.containers.map((container) =>
+    //   getContainerName(container, appConfig)
+    // );
+
+    const containerName = getContainerName(appConfig.containers[0], appConfig);
+
+    const runningContainers = await docker.listContainers();
+    // const isAppRunning = runningContainers.some((container) =>
+    //   containerNames.includes(container.Names[0].replace(/^\//, ''))
+    // );
+    const isAppRunning = runningContainers.some(
```

Comment:
this code shouldn't be on startApp. it should actually be done whenever Apps.tsx loads. so you can have a function inside madara-app.ts that fetches all running apps and the `Apps.tsx` component in react can call this function via the IPC